pkg/commands/git_commands/stash.go

```diff
@@ -49,6 +48,23 @@ func (self *StashCommands) ShowStashEntryCmdObj(index int) oscommands.ICmdObj {
 	return self.cmd.New(cmdStr).DontLog()
 }
 
+func (self *StashCommands) StashAndKeepIndex(message string) error {
+	return self.cmd.New(fmt.Sprintf("git stash save %s --keep-index", self.cmd.Quote(message))).Run()
+}
+
+func (self *StashCommands) StashUnstagedChanges(message string) error {
+	if err := self.cmd.New("git commit -m \"WIP\"").Run(); err != nil {
```

Comment:
We should use the --no-verify flag so that we skip hooks. I'd also change the message to something like 'WIP: lazygit stashing unstaged changes' so that when the user looks through their reflog they're not surprised to see the WIP commit in their reflog